include/ajax.tasks.php

```diff
@@ -424,7 +422,7 @@ function massProcess($action, $w=null)  {
 
             // Assume success
             if ($i==$count) {
-                $msg = sprintf(__('Successfully %s %s.'),
+                $msg = sprintf(__('Successfully %1$s %2$s.' /* Tokens are <actioned> <x selected task(s)> */),
```

Comment:
Could you possibly make this a standard comment like `//comment` and also move it outside the function (it looked like a parameter at first lol)?
I also think this comment is in more than one place..😄